src/components/ArticlePreview.jsx

```diff
@@ -48,13 +36,10 @@ export default class ArticlePreview extends React.Component {
 ArticlePreview.propTypes = {
   article: React.PropTypes.shape({
     title: React.PropTypes.string.isRequired,
-    image: React.PropTypes.string.isRequired,
-    slug: React.PropTypes.string.isRequired,
+    teaser: React.PropTypes.string.isRequired,
```

Comment:
Yeah I think we did. Emil's might be outdated because his branch was made before it was set.